apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/matcher/FastPathMatcher.java

```diff
@@ -84,6 +84,22 @@ private boolean normalMatch(String pat, int p, String str, int s) {
     private boolean wildcardMatch(String pat, int p, String str, int s) {
         char pc = safeCharAt(pat, p);
 
+        // End of pattern, check string only has zero or one '/' at end.
```

Comment:
If we get EOS(end of string) when we enter wildcard match state.
We just check the string only has zero or one '/' at end.